src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3AnnotationFactory.java

```diff
@@ -38,6 +38,9 @@ public class GFF3AnnotationFactory {
     ///  List of features that do not belong to a gene.
     List<GFF3Feature> nonGeneFeatures;
 
+    // Keeps track of the GFF3 ID given to EMBL features; Used to find the parent of a feature
+    // key: emblfeature_gene, value: gff3 ID
+    Map<String, String> emblIDtoGFF3ID;
```

Comment:
This won't work if the child is present before the parent feature. Can't we map the feature name back to the FF Feature using the same approach we set it to the Gff3 feature?